discovery-provider/src/solana/anchor_program_indexer.py

```diff
@@ -102,12 +113,14 @@ async def parse_tx(self, tx_sig):
         return {"tx_sig": tx_sig, "tx_metadata": tx_metadata, "result": None}
 
     def is_valid_instruction(self, parsed_instr: Dict):
+        # TODO find a better way to reconcile admin in mocks / default config
```

Comment:
still need this TODO?

Reply:
Yeah for now this doesn't block us but if we fully flesh this out and add more transaction types we'll run into this issue.